message ClaimTeam {
   required fixed32 id = 1;
}
